src/superphot_plus/telescopes.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import dataclass, field
+from typing import Dict
+
+import numpy as np
+from typing_extensions import Self
+
+from superphot_plus.priors.fitting_priors import MultibandPriors
+from superphot_plus.utils import get_band_extinctions
+
+@dataclass
+class Telescope:
+    """Holder for telescope-specific information."""
+
+    name: str = ""
+    priors: MultibandPriors = field(default_factory=MultibandPriors)
+    wavelengths: Dict[str, float] = field(default_factory=dict)
+
+    def __post_init__(self):
+        """Check that priors and wavelengths are defined for all filters."""
+        for b in self.priors.bands:
+            assert b in self.wavelengths
+            
+    def get_ordered_wavelengths(self):
+        """Return wavelengths in order that matches priors'
```

Comment:
For each of the public functions, can you add the parameters and returns section to the doc string comment?